book/src/README.md

```diff
@@ -0,0 +1,84 @@
+<img src="./images/amethyst_thumb.png" alt="Logo" width="96px" style="float:left;margin-right:15px"/>
+
+# The Amethyst Engine
+
+> Note: This project is a *work in progress* and is very incomplete. Pardon the
+> dust!
+
+Howdy! This book will teach you everything you need to know about building video
+games and interactive simulations with the Amethyst game engine. This engine is
+written entirely in [Rust][rs], a safe and fast systems programming language,
+and sports a clean and modern design. More correctly, though, Amethyst is
+actually a suite of separate libraries and tools that collectively make up a
+game engine.
+
+[rs]: https://www.rust-lang.org/
+
+Amethyst is free and open source software, distributed under a dual license of [MIT][ml]
```

Comment:
Now I understand mdBook this whole file is redundent. I probably should just delete it (it's still available in repo root).